CMakeLists.txt

```
@@ -416,6 +416,9 @@ endif ()
 # Unfortunately this configuration variable is necessary and will need to be
 # updated as time goes on and new versions are released.
 set(Boost_ADDITIONAL_VERSIONS
+
+  "1.74.0" "1.73"  
+  "17.4.0" "17.4"		
```

```suggestion
  "1.74.0" "1.74"  
  "17.3.0" "17.3"		
```

Okay, just a small change, removed the extra empty line at the top, you can just hit "commit suggestion" button.